atest/testdata/keywords/type_conversion/keyword_decorator_with_wraps.robot

```diff
@@ -0,0 +1,10 @@
+*** Settings ***
+Force Tags        require-py3
```

Comment:
`require-py3` isn't needed here, can be used, but is needed on the `atest/robot` side.